.github/policies/resourceManagement.yml

```diff
@@ -152,5 +152,18 @@ configuration:
       - addReply:
           reply: Hello! I noticed that you're targeting one of our servicing branches. Please consider updating the version.
       description: Comment on vs* branches
+    - if:
+      - payloadType: Pull_Request
+      - or:
+        - isAction:
+            action: Opened
+        - isAction:
+            action: Reopened
+      - filesMatchPattern:
+          pattern: ^.+\.swr$

```

Comment:
Technically, any changes to `MSBuild.VSSetup.*` projects could have a similar impact, as could _not_ updating the .swr file when an update is required. Perfectly reasonable as a best-effort reminder, though, thank you!

Reply:
Included this case into the condition.